index.html

```diff
@@ -73,27 +73,6 @@ <h3>Documentation</h3>
                     </a>
                 </div>
             </div>
-            <div class="col-md-4">
```

Comment:
It seems as though there's some unrelated work here that will need to be omitted from this PR.

Reply:
Sorry, This was unintentional.